packages/data/addon/adapters/facts/elide.ts

```diff
@@ -58,11 +60,17 @@ export default class ElideFactsAdapter extends EmberObject implements NaviFactAd
   private dataQueryFromRequest(request: RequestV2): string {
     const args = [];
     const { table, columns, sorts, limit, filters } = request;
-    const columnsStr = columns.map(col => getElideField(col.field, col.parameters)).join(' ');
+    const columnsStr = columns
+      .map(col => {
+        const fieldWithoutTable = REMOVE_TABLE_REGEX.exec(col.field)?.[1] || col.field;
```

Comment:
Aren't we doing this here: https://github.com/yahoo/navi/pull/976/files/c04c13a2a75b4c6a3a87d3b241585f066fd7e84e..72021ee34de8b8848f228c03d9351b585191c5e4#diff-6dbcee0ca8d4a66f7a25c10d07132fd9R35-R36

Reply:
The link isn't working. What line are you referring to?

Reply:
https://github.com/yahoo/navi/blob/72021ee34de8b8848f228c03d9351b585191c5e4/packages/data/addon/adapters/facts/elide.ts#L35-L36

Reply:
Missed that. Should be fixed now